apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import { Link } from 'react-router-dom';
+import { History } from 'history';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+
+import logo from '../../../assets/mull-logo.png';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebookSquare, faGoogle, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+import './login.scss';
+
+export interface LoginProps {
+  history: History;
+}
+
+export const Login = ({ history }: LoginProps) => {
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+
+    validationSchema: Yup.object({
+      email: Yup.string().required('Email is required.').email('Email format is incorrect'),
+      password: Yup.string().required('Password is required.'),
+    }),
+
+    onSubmit: (values) => {},
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <div className="login-container">
+        <img src={logo} className="login-mull-logo" alt="Mull logo" />
+
+        <CustomTextInput
+          title="Email"
+          fieldName="email"
+          value={formik.values.email}
+          onChange={formik.handleChange}
+          hasErrors={formik.touched.email && !!formik.errors.email}
+          errorMessage={formik.errors.email}
+        />
+
+        <CustomTextInput
+          title="Password"
+          fieldName="password"
+          value={formik.values.password}
+          onChange={formik.handleChange}
+          hasErrors={formik.touched.password && !!formik.errors.password}
+          errorMessage={formik.errors.password}
+          password
+        />
+
+        <button type="submit" className="login">
+          Login
+        </button>
+
+        <div className="sign-up">
+          Don't have an account?{' '}
+          <Link to="/register" className="register-login-page">
+            {' '}
+            Sign up here!
```

Comment:
You insert two spaces in two places. I think its possible to insert both in one place
```suggestion
          Don't have an account?{'  '}
          <Link to="/register" className="register-login-page">
            Sign up here!
```